### PR TITLE
rgw_sal_motr: [CORTX-29873] - improve logging in rgw sal

### PR DESCRIPTION
**Pull request by sjain09**
### Problem
RGW SAL layer does not print every rc return from libmotr calls in case of error.
This sometimes makes it difficult to debug any issue.

### Solution
Add more logs in order to print rc for every failure received from libmotr in rgw sal.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by sumedhak27 — APPROVED** [review]

File: src/rgw/rgw_sal_motr.cc [hunk 1]
```diff
@@ -4135,7 +4135,7 @@ int MotrStore::delete_motr_idx_by_name(string iname)
   struct m0_uint128 idx_id;
   struct m0_op *op = nullptr;
 
-  ldout(cctx, 20) << __func__ << ": delete_motr_idx_by_name=" << iname << dendl;
+  ldout(cctx, 20) << __func__ << ": iname=" << iname << dendl;
```
Comment (sumedhak27) [inline src/rgw/rgw_sal_motr.cc:4138]:
```suggestion
  ldout(cctx, 20) << __func__ << ": iname = " << iname << dendl;
```

File: src/rgw/rgw_sal_motr.cc [hunk 2]
```diff
@@ -166,7 +166,7 @@ int MotrMetaCache::remove(const DoutPrefixProvider *dpp,
   ObjectCacheInfo info;
   int rc = distribute_cache(dpp, name, info, INVALIDATE_OBJ);
   if (rc < 0) {
-    ldpp_dout(dpp, 0) << "ERROR: " <<__func__<< "(): failed to distribute cache: rc =" << rc << dendl;
+    ldpp_dout(dpp, 0) << "ERROR: " <<__func__<< ": failed to distribute cache: rc = " << rc << dendl;
```
Comment (sumedhak27) [inline src/rgw/rgw_sal_motr.cc:169]:
```suggestion
    ldpp_dout(dpp, 0) << "ERROR: " << __func__ << ": failed to distribute cache: rc = " << rc << dendl;
```

**Review by dharmendra-jyani — APPROVED** [review]
Approved

File: src/rgw/rgw_sal_motr.cc (hunk 2 above)
Comment (sjain09) [reply src/rgw/rgw_sal_motr.cc:169]: For now I going ahead. this change can be taken later.

**Pull request merged**